app/redux/Store.js

```diff
@@ -62,7 +62,7 @@ export default initializeStore = () => {
             createFilter('history', ['viewedCounter']),
             createFilter('appState', ['multiTaskCounter', 'lockedCounter']),
             createFilter('connectedDevice', ['numDisconnects', 'numReconnects']),
-            createFilter('analysis', ['e1RMExercise', 'e1RMVelocity', 'e1RMDaysRange', 'tagsToInclude', 'tagsToExclude']),
+            createFilter('analysis', ['e1RMExercise', 'e1RMVelocity', 'velocitySlider', 'e1RMDaysRange', 'tagsToInclude', 'tagsToExclude', 'e1rm', 'confidence']),
```

Comment:
- [ ] What does each property do? Why must we save them? Theoretically if some of these values are calculated, we don't need to store them as they can be recalculated

Reply:
So we don't want to store previous 1rm calculations? This is more of a product level question, I feel it's good to show something rather than ---. We can discuss further.